crates/storage/trie/trie.rs

```diff
@@ -507,9 +525,10 @@ mod test {
         );
     }
 
+    #[cfg(feature = "libmdbx")]
```

Comment:
Not necessary, the default feature is "libmdbx" for each crate making use of it. If we run the tests using `--no-default-features` it will not compile `libmdbx` and will run just `in_memory` tests.